.circleci/config.yml

```diff
@@ -115,6 +115,8 @@ jobs:
           root: .
           paths:
             - docs
+            - docs/node_modules
```

Comment:
Are these required? Doesn't just `docs` cover both of these?

Reply:
It shouldn't be required. It was failing before with `sh: 1: gatsby: not found` without it and the `node_modules` wasn't being persisted. It looks to be working now.